### PR TITLE
64 add in core functional components

### LINKED ISSUE #64: Add in core functional components
References #47

### PR DESCRIPTION
**Pull request by javitrejo14**
Closes #64

### CODE REVIEW

File: src/mainPage.html [hunk 1]
```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (PTLux1517) [inline src/mainPage.html:2]: Now that the main page elements have been added to App.tsx, I imagine we won't need this standalone html file. If you added it to show the work you did along the way, it'll still be there in your series of commits and in this review, to prove you did write it, even if it doesn't make it into the main branch. If we wanted to keep it though as our prototype, we could move it into its own `prototype` directory apart from `src`.

File: src/upload_button.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+    return e(
+      'button',
+      { onClick: () => this.setState({ clicked: true }) },
+      'Choose Marker Data File'
+    );
+  }
+}
+
+const domContainer = document.querySelector('#upload_button_container');
```
Comment (PTLux1517) [inline src/upload_button.js:24]: This file appears to be connected with `src/mainPage.html`, so the same idea as above applies.

**Review by PTLux1517 — CHANGES_REQUESTED** [review]

File: src/mainPage.html (hunk 1 above)
Comment (PTLux1517) [reply src/mainPage.html:2]: I'm open to other team members' thoughts too if anyone wants to chime in.

Comment (javitrejo14) [reply src/mainPage.html:2]: Yeah I knew it wouldn't go into the main branch, I realized it was incorrect after I created the pull request. I just didn't know how to go about removing it without completely deleting the work I did / time I put in.

**Review by vpzomtrrfrt — CHANGES_REQUESTED** [review]
I'd rather not commit font files directly into the repo, can we use a package like https://www.npmjs.com/package/@fontsource/goldman instead?

**Review by wkenny80 — APPROVED** [review]

**Pull request closed without merging**